.buildkite/browser-pipeline.yml

```diff
@@ -4,45 +4,45 @@ agents:
 steps:
   - group: "Browser Tests"
     steps:
-      - label: ":docker: Build BitBar Maze Runner image${EXTRA_STEP_LABEL}"
-        key: "browser-maze-runner-bb-${USE_CDN_BUILD}"
-        timeout_in_minutes: 20
-        plugins:
-          - docker-compose#v4.12.0:
-              build: browser-maze-runner-bb
-              image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
-              cache-from: browser-maze-runner-bb:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:performance-ci-browser-${BRANCH_NAME}-${USE_CDN_BUILD}
-          - docker-compose#v4.12.0:
-              push: browser-maze-runner-bb:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:performance-ci-browser-${BRANCH_NAME}-${USE_CDN_BUILD}
-        env:
-          USE_CDN_BUILD: "${USE_CDN_BUILD}"
-        retry:
-          automatic:
-            - exit_status: "*"
-              limit: 1
 
-      - label: ":docker: Build BrowserStack Maze Runner image${EXTRA_STEP_LABEL}"
-        key: "browser-maze-runner-bs-${USE_CDN_BUILD}"
+      - label: "Build browser fixtures - ${BUILD_MODE}"
+        key: "build-browser-fixtures-${BUILD_MODE}"
         timeout_in_minutes: 20
-        plugins:
-          - docker-compose#v4.12.0:
-              build: browser-maze-runner-bs
-              image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
-              cache-from: browser-maze-runner-bs:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:performance-ci-browser-${BRANCH_NAME}-${USE_CDN_BUILD}
-          - docker-compose#v4.12.0:
-              push: browser-maze-runner-bs:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:performance-ci-browser-${BRANCH_NAME}-${USE_CDN_BUILD}
+        agents:
+          queue: macos-14
         env:
-          USE_CDN_BUILD: "${USE_CDN_BUILD}"
+          BUILD_MODE: "${BUILD_MODE}"
```

Comment:
There's a problem here when the pipeline is triggered based on a file change.  I need to have that script set the environment variable when uploading the pipeline file.